client/public/index.html

```diff
@@ -14,7 +14,7 @@
       manifest.json provides metadata used when your web app is installed on a
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <!-- <link rel="manifest" href="%PUBLIC_URL%/manifest.json" /> -->
```

Comment:
Please undo the comment on this line

Reply:
Thanks for checking. I undo the comment on <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />